packages/arb-token-bridge-ui/src/components/TransactionHistory/TransactionStatusInfo.tsx

```diff
@@ -45,7 +45,7 @@ export const TransactionStatusInfo = ({
     return null
 
   return (
-    <div className="flex items-center gap-1 rounded-md bg-orange p-2 text-base text-brick-dark lg:flex-nowrap">
+    <div className="flex items-center gap-1 rounded-md bg-orange p-2 text-base text-brick-dark lg:flex-nowrap mx-2">
```

Comment:
This margin was not intended for non-mobile screens. Pls update it to `lg:mx-2`.

<img width="1220" alt="image" src="https://user-images.githubusercontent.com/7558499/229606750-91dbaa19-ab30-4c23-8f3c-0737afdee129.png">

Reply:
For mobile-only styling as required, the style should be `mx-2 lg:mx-0` instead of `mx-2`. 
Also, the current styling is not prettier compatible. Please install prettier and format the document before saving it.

<img width="756" alt="image" src="https://user-images.githubusercontent.com/7558499/229735434-2dd57998-f32a-420b-bf1a-fd04a583d601.png">